src/uu/more/src/more.rs

```diff
@@ -264,54 +270,52 @@ struct Pager<'a> {
     // The current line at the top of the screen
     upper_mark: usize,
     // The number of rows that fit on the screen
-    content_rows: usize,
+    content_rows: u16,
     lines: Vec<String>,
     next_file: Option<&'a str>,
     line_count: usize,
-    close_on_down: bool,
     silent: bool,
 }
 
 impl<'a> Pager<'a> {
-    fn new(rows: usize, lines: Vec<String>, next_file: Option<&'a str>, silent: bool) -> Self {
+    fn new(rows: u16, lines: Vec<String>, next_file: Option<&'a str>, silent: bool) -> Self {
```

Comment:
I find having the rows as `u16` to be a bit low. I can imagine giant log files where you might want to check out some stuff near the end that exceed 65000 lines (really only useful once we implement jumping to specific lines)

Reply:
But this is the size of the terminal tho. I highly doubt anyone having terminals larger than `u16` rows or columns. I didn't change the type of `upper_mark` which is what we used to index the `lines`.

Reply:
Ah, you're right!